tardis/simulation/tests/test_simulation.py

```diff
@@ -172,3 +173,95 @@ def test_logging_simulation(atomic_data_fname, caplog):
 
     for record in caplog.records:
         assert record.levelno >= logging.INFO
+
+
+# Testing Configuration of Logger via run_tardis() Function
+# TODO: Move to its own Testing Module (Logger)
+@pytest.mark.parametrize(
+    ["log_state", "specific"],
+    [
+        ("Info", False),
```

Comment:
why are we testing different cases?

Reply:
upper/lower cases

Reply:
Yes absolutely. I was testing for different cases for the input values. I will add an expected failure tests for parameters if they are not valid. That way we could test for failures as well.

Reply:
this is not what I meant. ultimately this is not something that you should test, because it should be tested by the string class in python. Users put it in in any form they like and you put it to a specific form (my suggestion `"DeBuG".lower()`) to the form that you are testing against.